jbmc/regression/jbmc/class-loading-annotations/test-class-value-annotation-on-class.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+ClassValueAnnotationOnClass.class
+--verbosity 10
+Getting class `MyClassA' from file \.[\\/]MyClassA\.class
+--
+Getting class `MyClassB' from file \.[\\/]MyClassB\.class
```

Comment:
:pick: :bulb: In general I try to be more cautious about the regexes in the non-matching part (because a single type could make the test useless). Consider replacing with `Getting class 'MyClassB'`.

Reply:
Good point. Replaced the non-matching regexes with just `MyClassB`.